sal_interfaces/MTM2/MTM2_Telemetry.xml

```diff
@@ -51,6 +51,54 @@
     </item>
 </SALTelemetry>
 
+<SALTelemetry>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_mirrorPositionMeasuredIMS</EFDB_Topic>
```

Comment:
I suggest renaming this topic to `positionIMS`.

Reply:
I think MTM2_mirrorPositionMeasuredIMS is fine, since we already have MTM2_mirrorPositionMeasured

Reply:
This name is based on the LTS-162 v5.

Reply:
So, I think it is already implicit that this is a "mirror" position because the component is `M2`... I'm thinking in terms of someone looking at the interface and trying to build a query for it... it is much simpler and easy to understand `positionIMS` then  `mirrorPositionMeasuredIMS`... 

Most of my further comments are on those lines as well...